docs/topic_guide/installation.rst

```diff
@@ -0,0 +1,102 @@
+.. _aiapy-topic-guide-installing:
+
+************
+Installation
```

Comment:
Why is this repeated. Shouldn't we just direct people to the SunPy installation guide, especially for things like installing Python?

Reply:
Duplicated to keep the flow of the docs more internal but I can do this.